src/lando/main/scm/__init__.py

```diff
@@ -0,0 +1,10 @@
+# flake8: noqa
+from .abstract_scm import *
+from .consts import *
+from .exceptions import *
+from .hg import *
```

Comment:
Although we do this in `main.models`, it is conventionally discouraged to do this elsewhere (the models implementation being largely due to legacy porting, and since it is fairly self-contained). I think it would be better to just be explicit with the imports where they happen instead.

Reply:
Ah, that's a pity. I find dot-imports to be convenient _within the same module_ (no `..`) to allow to keep the structure itself modular without hardcoding the full namespace, which makes it somewhat easier to move the module if need be.

I'll fix.